src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -5,98 +5,521 @@
 
-    def test_forcedelete(self):
 
-        managed_cluster_name = self.create_random_name(prefix='test-force-delete', length=24)
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='conk8stest', location='eastus2euap', random_name_length=16)
+    def test_upgrade(self,resource_group):
+
+        managed_cluster_name = self.create_random_name(prefix='test-upgrade', length=24)
         kubeconfig="%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')) 
         self.kwargs.update({
             'name': self.create_random_name(prefix='cc-', length=12),
+            'rg': resource_group,
             'kubeconfig': kubeconfig,
-            # 'kubeconfig': "%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')),
             'managed_cluster_name': managed_cluster_name
         })
 
-        self.cmd('aks create -g rohanazuregroup -n {} -s Standard_B4ms -l westeurope -c 1 --generate-ssh-keys'.format(managed_cluster_name))
-        self.cmd('aks get-credentials -g rohanazuregroup -n {managed_cluster_name} -f {kubeconfig}')
-        self.cmd('connectedk8s connect -g rohanazuregroup -n {name} -l eastus --tags foo=doo --kube-config {kubeconfig}', checks=[
+        self.cmd('aks create -g {rg} -n {managed_cluster_name} --generate-ssh-keys')
+        self.cmd('aks get-credentials -g {rg} -n {managed_cluster_name} -f {kubeconfig}')
+
+        self.cmd('connectedk8s connect -g {rg} -n {name} -l eastus --tags foo=doo --kube-config {kubeconfig} --kube-context {managed_cluster_name}', checks=[
             self.check('tags.foo', 'doo'),
             self.check('name', '{name}')
         ])
-        self.cmd('connectedk8s show -g rohanazuregroup -n {name}', checks=[
+        self.cmd('connectedk8s show -g {rg} -n {name}', checks=[
             self.check('name', '{name}'),
-            self.check('resourceGroup', 'rohanazuregroup'),
+            self.check('resourceGroup', '{rg}'),
             self.check('tags.foo', 'doo')
         ])
 
-        # Simulating the condition in which the azure-arc namespace got deleted
-        # connectedk8s delete command fails in this case
-        subprocess.run(["kubectl", "delete", "namespace", "azure-arc","--kube-config", kubeconfig])
+        os.environ.setdefault('KUBECONFIG', kubeconfig)
+        helm_client_location = install_helm_client()
+        cmd = [helm_client_location, 'get', 'values', 'azure-arc', "--namespace", "azure-arc-release", "-ojson"]
 
-        # Using the force delete command
-        # -y to supress the prompts
-        self.cmd('connectedk8s delete -g rohanazuregroup -n {name} --kube-config {kubeconfig} --force -y')
-        self.cmd('aks delete -g rohanazuregroup -n {} -y'.format(managed_cluster_name))
+        # scenario - auto-upgrade is true , so implicit upgrade commands dont work
+        self.cmd('connectedk8s update -n {name} -g {rg} --auto-upgrade true --kube-config {kubeconfig} --kube-context {managed_cluster_name}')
```

Comment:
should this be upgrade command?  why are we running update command with autoupgrade true?

Reply:
based on the scenario description, it looks like it should be upgrade command